aries-site/src/examples/templates/wizard/StickyHeaderFooterExample.js

```diff
@@ -140,14 +140,28 @@ export const StickyHeaderFooterExample = () => {
   // controls state of cancel layer
   const [open, setOpen] = useState(false);
   const size = useContext(ResponsiveContext);
+
+  const wizardRef = React.useRef();
+
```

Comment:
can we add some comments here like above ?

Reply:
Yes, I can add some comments.